Notepad/HomeViewController.swift

```diff
@@ -12,25 +12,38 @@ class HomeViewController: UIViewController {
     @IBOutlet var bodyTextView: UITextView!
     @IBOutlet var titleTextField: UITextField!
     
+    let title_key = "Title"
+    let body_key = "Body"
     override func viewDidLoad() {
         super.viewDidLoad()
     }
     
     @IBAction func handleSaveButtonPressed(_ sender: Any) {
-        let title = titleTextField.text
-        print("my title is: \(title!)")
         
-        let notes = bodyTextView.text
-        print("my body is: \(notes!)")
         
-        UserDefaults.standard.set(title, forKey: "Title")
-        UserDefaults.standard.set(notes, forKey: "Body")
+        let alert = UIAlertController(title: " ", message: "Would you like to save your note?", preferredStyle: .alert)
```

Comment:
What other styles are there besides `.alert`?

Reply:
what style the variable name?

Reply:
Try to switch it to that, and see how the UI changes

Reply:
it crashed cuz it needed location information or smth according to the error message